requirements-extra.txt

```diff
@@ -5,6 +5,5 @@ psutil>=4.0.0
 lz4>=1.0.0
 protobuf>=3.6
 gevent>=1.3.0
-etcd-gevent>=0.1.0
```

Comment:
Is there any error messages about etcd-gevent so we can fix?

Reply:
I collect the feedback from users that the dependency like `geventhttpclient` is hard to install.

Reply:
Source packages are always hard to install. Okay, forget that.